Riku/src/GameLogic/Assets/RandomWrapper.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <cstdlib>
+
+class RandomWrapper
+{
+public:
+	RandomWrapper() {};
+	int generate(int x, int y)
+	{
+		return std::rand() % (y - x) + x;
+	}
+};
+
```

Comment:
Jeśli chodzi o tą klasę to jeszcze nie jest używana, ale może być potrzebna do generowania liczb losowych w skrypcie tworzącym mapę. Na ten moment nie udało mi się tego osiągnąć